src/main/resources/i18n/Execution_de.properties

```diff
@@ -0,0 +1,8 @@
+#
+# This resource bundle is used for the query execution code to produce i18n messages
+#
+# REMEMBER - a single quote ' in MessageFormat means things that are never replaced within them
+# so use 2 '' characters to make it one ' on output.  This will take for the form ''{0}''
+#
+Execution.handleOneOfNotOneFieldError=Es muss genau ein Schl�ssel angegeben werden f�r OneOf Typ ''{0}''.
+Execution.handleOneOfValueIsNullError=OneOf type field ''{0}'' darf nicht null sein.
```

Comment:
A bit unsure about `type field` but I left it the same in the Dutch translations

Reply:
It's so hard to tell when to use German or English computer science words!

We do have phrases like "Fehlendes directive argument" in other strings so this would be consistent

Reply:
That should be 

```
Es muss genau ein Key angegeben werden für OneOf Typ ''{0}''.
```